### PR TITLE
Updating fluentd docker container mount path

### PR DESCRIPTION
**Pull request by ewolinetz**
To address https://trello.com/c/V5osNVuq/244-free-int-fluentd-not-running-due-to-var-lib-docker-containers-issue

cc @jupierce

### CODE REVIEW

**Comment by ewolinetz** [comment]
/hold

This may not be something we actually need to change. The fix that this patches should actually be resolved at the lower level. Rich is digging up the discussion that was had surrounding this in a prior email thread.

**Comment by ewolinetz** [comment]
http://post-office.corp.redhat.com/archives/aos-devel/2018-April/msg00231.html

**Comment by ewolinetz** [comment]
cc @eparis @gnufied @jsafrane @vikram-redhat

File: roles/openshift_logging_fluentd/templates/2.x/fluentd.j2 [hunk 1]
```diff
@@ -61,7 +61,7 @@ spec:
         - name: varlog
           mountPath: /var/log
         - name: varlibdockercontainers
-          mountPath: /var/lib/docker/containers
+          mountPath: /var/lib/docker
           readOnly: true
```
Comment (gnufied) [inline roles/openshift_logging_fluentd/templates/2.x/fluentd.j2]: Does fluentd pod works with either /var/lib/docker or /var/lib/docker/containers without any modification? I would expect these 2 directories to contain different content.

**Comment by gnufied** [comment]
My main concern same as before is - this will again break in future if docker deployment causes "/var/lib/docker" to be mounted as private mount.

Rather than doing this here - is it possible to disable `MountPropagation` feature from inventory file that deploys the cluster to free-int? I tried making the same change in openshift-ansible - https://github.com/openshift/openshift-ansible/pull/7936 but now that I think of, since we are rolling back "slave" mount propagation defaults in upstream, modifying openshift-ansible may not be the right place.

**Comment by gnufied** [comment]
Any change we make in openshift-ansible will be temporary(and not required) to fix this issue once https://github.com/openshift/origin/pull/19364 PR merges. So it is best to workaround this by disabling `MountPropagation` feature via inventory file.

**Comment by jsafrane** [comment]
https://github.com/openshift/origin/pull/19364 was merged into Origin 3.10, so this PR is probably not needed. Sorry it took so long.

**Comment by ewolinetz** [comment]
Closing as the origin fix for this has merged

**Pull request closed without merging**